contribs/gnodev/Makefile

```diff
@@ -5,11 +5,13 @@ GOTEST_FLAGS ?= $(GOBUILD_FLAGS) -v -p 1 -timeout=5m
 rundep := go run -modfile ../../misc/devdeps/go.mod
 golangci_lint := $(rundep) github.com/golangci/golangci-lint/cmd/golangci-lint
 
-install:
+install: install.gnodev
```

Comment:
Why not install gnobro by default with gnodev?

Reply:
I discussed a bit with @moul about this, and we think that it's better to keep the installation manual for now. We don't want to install every binary by default when you run `make install` at the root folder.